src/compiler/transformers/ts.ts

```diff
@@ -1696,8 +1697,9 @@ export function transformTypeScript(context: TransformationContext) {
     }
 
     function visitParenthesizedExpression(node: ParenthesizedExpression): Expression {
-        const innerExpression = skipOuterExpressions(node.expression, ~OuterExpressionKinds.Assertions);
-        if (isAssertionExpression(innerExpression)) {
+        const kinds = OuterExpressionKinds.Assertions | OuterExpressionKinds.SatisfiesExpressions;
+        const innerExpression = skipOuterExpressions(node.expression, ~kinds);

```

Comment:
nit: I think this would be slightly more in line with the overall style in the codebase
```suggestion
        const innerExpression = skipOuterExpressions(node.expression, ~(OuterExpressionKinds.Assertions | OuterExpressionKinds.SatisfiesExpressions));
```